graylog2-web-interface/src/views/components/widgets/MessageList.jsx

```diff
@@ -42,35 +42,39 @@ type State = {
 }
 
 type Props = {
-  fields: FieldTypeMappingsList,
-  pageSize: number,
   config: MessagesWidgetConfig,
+    currentView: {
+      activeQuery: string,
+      view: {
+        id: number,
+      },
+    },
   data: { messages: Array<Object>, total: number, id: string },
-  selectedFields?: Immutable.Set<string>,
+  editing: boolean,
+  fields: FieldTypeMappingsList,
+  onConfigChange: (MessagesWidgetConfig) => Promise<void>,
```

Comment:
I've created a `onSortChange` method inside the `MessageList`, so `FieldSortIcon` does not need to know how to change the config. But I did not changed `onConfigChange` in `Widget.jsx` because all widgets are receiving this function and I thought onSortChange` would be too specific.